### DISCUSSION TITLE
How to solve these ploblem when I update 0.10 to version 11?

### DISCUSSION THREAD

**Discussion by wangkkk** [AvaloniaUI/Avalonia#14032] [Q&A]
I meet these ploblem when update 0.10 to 11.Use what method to Replace it?Thanks.
![QQ图片20231208090033](https://github.com/AvaloniaUI/Avalonia/assets/4760868/cc2aa6ce-62f2-4243-a85e-8bd350fd3305)
![QQ图片20231208090051](https://github.com/AvaloniaUI/Avalonia/assets/4760868/48ffc970-9659-4c0e-bdc7-8095152e901d)
![QQ图片20231208091341](https://github.com/AvaloniaUI/Avalonia/assets/4760868/78a2c151-622b-451a-8bbe-3e068cbe5afb)
![QQ图片20231208091348](https://github.com/AvaloniaUI/Avalonia/assets/4760868/5f4f7dc5-ba14-4d16-9652-28e38d704fa9)

**Comment by maxkatz6** [comment]
Do you need these for testing? If so, check our documentation on Headless testing.
